weave/weave_client.py

```diff
@@ -478,7 +478,9 @@ def create_call(
         return call
 
     @trace_sentry.global_trace_sentry.watch()
-    def finish_call(self, call: Call, output: Any) -> None:
+    def finish_call(
```

Comment:
This file is a simple refactor with no logical changes other than the ability to have both an output and an exception now